File: ruby/raindrops/raindrops.rb
class Raindrops

    def self.convert(num)
        response = ""
        if num%3==0
            response << "Pling"
        end
        if num%5==0
            response << "Plang"
        end
        if num%7==0
            response << "Plong"
        end
        if num%3 != 0 && num%5 != 0 && num%7 != 0
            return num.to_s
        else
            return response
        end
    end

end

File: python/raindrops/raindrops.py
def convert(number):
    my_string = ""
    if number%3==0:
        my_string = my_string + "Pling"
    if number%5==0:
        my_string = my_string + "Plang"
    if number%7==0:
        my_string = my_string + "Plong"
    if len(my_string) > 0:
        return my_string
    else:
        return str(number)